KSPRC-CityLights

```diff
@@ -0,0 +1,24 @@
+{
+	"spec_version": "v1.0",
+	"identifier":   "KSPRC-CityLights",
+	"name":         "KSPRC city lights",
+	"abstract":     "Standalone city lights from KSPRC.",
+	"$kref":        "#/ckan/spacedock/690",
+	"ksp_version":  "any",
```

Comment:
Hmm, SpaceDock and the forum thread both specify KSP 1.1.2. Are we sure `"any"` is kosher?

![image](https://user-images.githubusercontent.com/1559108/81756069-4d30f400-9480-11ea-8637-3a4038e0f624.png)

![image](https://user-images.githubusercontent.com/1559108/81756050-40140500-9480-11ea-8b96-c65381c02407.png)

Reply:
Eh... sort of sure. This will be compatible with all KSP versions that support EnvironmentalVisualEnhancements, and it first released for 1.1.2. I was under the mindset that marking it as compatible for all modes, and requiring EVE would make it only work for all EVE versions. Is that good? 
![image](https://user-images.githubusercontent.com/5304201/81756169-d986eb00-946f-11ea-8193-375caccc2342.png)
There is a mod in CKAN that's simply the KSPRC textures which is also marked as 'any', and I was using that as an example since they both draw from the same source.

By the way thank you so much for all your help, and I hope my edits haven't been too annoying. I'm not exactly sure how to properly add a file change to the master so I've resorted to forking it and working from there. I think I'm getting the hang of it now, though.

Reply:
> EnvironmentalVisualEnhancements which first released for 1.1.2

Nah, we have a version indexed for KSP 0.25-1.0.4:

- https://github.com/KSP-CKAN/CKAN-meta/blob/master/EnvironmentalVisualEnhancements/EnvironmentalVisualEnhancements-7-4.ckan

Reply:
Ah! That version is probably the original that was created by rbray before Waz took over when he left. This is not compatible for that.

Reply:
> probably causing you a bit of a headache

Oh no, no headache whatsoever. You're keeping my brain active, like doing crossword puzzles. :grin: 

> I should fork the master, make my edits to the files I want in my fork, and then if it looks good I click this and give it a useful title

You've already got a fork for the Spectra changes, so you can skip this part. Click this link, click the pencil icon, then make your changes and commit them:

- https://github.com/Avera9eJoe/NetKAN/blob/patch-1/NetKAN/Spectra.netkan